runners/google-cloud-dataflow-java/worker/src/main/java/org/apache/beam/runners/dataflow/worker/streaming/config/StreamingEngineComputationConfigFetcher.java

```diff
@@ -187,6 +194,18 @@ private static StreamingEnginePipelineConfig createPipelineConfig(StreamingConfi
       pipelineConfig.setMaxWorkItemCommitBytes(config.getMaxWorkItemCommitBytes().intValue());
     }
 
+    if (shouldPerformOutputSizeChecks && config.getOperationalLimits() != null) {
```

Comment:
seems like we could remove the shouldPerformOutputSizeChecks and just perform them if the limit is not the max if getOperatoinalLimits returns a default of max values

Reply:
I was looking for some way to only enable this feature by experiment first. `shouldPerformOutputSizeChecks` is passed in based on an experiment being set. If you think that's overkill, then yeah, I can just remove that parameter entirely. WDYT?

Reply:
that sounds good, I misread this earlier and thought it was where we were evaluating limits.

Do you want to just add the experiment usage in this PR?

Reply:
I did. It's hidden in StreamingDataflowWorker.java where we call           `StreamingEngineComputationConfigFetcher.create()`.

Reply:
Not sure what we want to call the experiment, but I went with "throw_exceptions_on_large_output".

Reply:
Thinking about this some more, maybe we want to plumb this further rather than just setting the limits to the max.

If we kept the limits, we could check them and at least log an error message if we weren't going to throw an exception.

Reply:
Ok. I'll go ahead and add `shouldThrowException...` to OperationalLimits so we can decide what we want to do anywhere we're actually checking the limits. This should make cleanup easier when we decide to get rid of the experiment and make it always-on.